tools/clone_flutter.sh

```diff
@@ -33,10 +33,25 @@ fi
 LATEST_COMMIT_TIME_ENGINE=`git log -1 --date=local --format="%cd"`
 echo "Latest commit time on engine found as $LATEST_COMMIT_TIME_ENGINE"
 
-# Do rest of the task in the root directory
-cd ~
-mkdir -p $FRAMEWORK_PATH
-cd $FRAMEWORK_PATH
+# Check if there is an argument added for repo location.
```

Comment:
(Github doesn't let me comment on unchanged lines; this comment is about lines 5-9)

If we are going to use `felt` for running integration tests locally, the expectation is that if a test is broken on CI it should break in the exact same way locally. Therefore, we should use this same script when running tests that use the framework locally as well. We should therefore remove the limitation that this script is only used on CI and actually use it when testing locally.

Reply:
/cc @tvolkert @cbracken to comment on mobile bits.

Reply:
I think we reviewed this file together. We even tried to make sure we added many check so that it is not used locally. Can you explain why the reasoning changed and why we are removing a constraint we added so strictly at that time?

Reply:
The reasoning changes because we are enabling running integration tests locally via `felt` and adding more tests. Let's enumerate the reasons why you might want to run tests locally:

* You are working on code and want to make sure tests pass locally before submitting a PR. Your expectation is that if a test passes locally it will also pass on CI.
* A breaking change was committed to master and you want to investigate it. You checkout the broken commit and run tests locally. Your expectation is that you will get the exact same error locally.

For this to work reliably you want to be sure of two things:

* The versions of the engine and the framework you are using locally are exactly the same as those that were used on the CI.
* You are using exactly one version of the engine and exactly one version of the framework.

If any one of these things does not hold you end up in bad situations, such as inability to reproduce CI failures, flakiness, confusion.

Reply:
done.